extensions/objects/templates/FilepathEditor.js

```diff
@@ -73,6 +75,21 @@ oppia.directive('filepathEditor', [
                  data.metadata.savedUrl.length > 0;
         };
 
+        $scope.getImageSizeHelp = function() {
+          var imageWidth = $scope.data.metadata.originalWidth;
+          if ($scope.resizeRatio === 1 &&
+              imageWidth > $scope.MAX_OUTPUT_IMAGE_WIDTH) {
+            return 'Even at 100%, the image is being automatically downsized ' +
```

Comment:
Sure.

I was going back and forth for a moment with this message. At some point while using it, I thought the message only makes sense when the img is at 100%, and it's only at that point that the user may think "why is it saying 490px? this img is much bigger" ... that's also why the tooltips doesn't even appear if resize < 100%, cause in that case the user knows why the image is getting small (because they'reasking for it).

But anyway, the message looks fine to me either way.